pkgs/development/python-modules/wandb/default.nix

```diff
@@ -118,6 +119,23 @@ let
 
     sourceRoot = "${src.name}/core";
 
+    # x86_64-darwin fails with:
+    # "link: duplicated definition of symbol dlopen, from github.com/ebitengine/purego and github.com/ebitengine/purego"
+    # This is fixed in purego 0.8.3, but wandb-core uses 0.8.2, so we pull in the fix here.
+    patches = lib.optionals (stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isx86_64) [
+      (fetchpatch {
```

Comment:
fetchpatch2 is now preferred 

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#fetching-patches

Reply:
Fixed